### PR TITLE
Post-mortem: Degraded Host Network

### PR DESCRIPTION
**Pull request by maxmcd**

### CODE REVIEW

File: src/content/blog/post-mortem-degraded-host-network.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+pubDate: Dec 31, 2024
+author: Max McDonnell
+---
+
+At 12:15am ET one of our servers started to experience long timeouts on both
+inbound and outbound connections. Existing connections continued to serve
+requests and some inbound and outbound requests were successful, but most
+requests would stall indefinitely in a pending state, timing out without
+establishing a connection. This persisted until 7:04am ET when we manually
+terminated the server after a user report.
```
Comment (zephraph) [inline src/content/blog/post-mortem-degraded-host-network.md:14]: Should we mention the network degradation here?

Comment (maxmcd) [reply src/content/blog/post-mortem-degraded-host-network.md]: @zephraph updated, lmk what you think

**Comment by stevekrouse** [comment]
Looks great to me! I think totally shippable. I think it might read a little better with subheadings, ie "timeline", "impact", "next steps" etc. I think this one has a good structure: https://blog.val.town/blog/post-mortem-blob-storage-outage/

**Comment by maxmcd** [comment]
Sweet, merging. 

@zephraph if you have further feedback I'm happy to handle that in a follow-on.

**Pull request merged**